### PR TITLE
change rand to fastrand

### PR DESCRIPTION
**Pull request by qiangmzsx**
randLB change rand to fastrand

### CODE REVIEW

**Comment by qiangmzsx** [comment]
change rand to fastrand update vendor

**Comment by qiangmzsx** [comment]
remove lb init func and .idea

File: pkg/util/ip_util.go [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package util
+
+import (
+	"strings"
+
+	"github.com/valyala/fasthttp"
+)
+
+//获取真实的
```
Comment (zhangxu19830126) [inline pkg/util/ip_util.go]: Use go comment style

File: pkg/lb/haship.go [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+	return lb
+}
+
+func (haship HashIPBalance) Select(ctx *fasthttp.RequestCtx, servers []metapb.Server) uint64 {
+	size := len(servers)
+	if size < 1 {
+		return 0
+	}
+	hash := fnv.New32a()
+	//key为客户端ip
```
Comment (zhangxu19830126) [inline pkg/lb/haship.go]: Use english to comment

**Comment by qiangmzsx** [comment]
Use english to comment

File: pkg/lb/haship.go (hunk 2 above)
Comment (qiangmzsx) [reply pkg/lb/haship.go]: over

File: pkg/util/ip_util.go (hunk 1 above)
Comment (qiangmzsx) [reply pkg/util/ip_util.go]: over

**Comment by qiangmzsx** [comment]
Modify the comment to match golint

**Pull request merged**